bin/check-opsgenie-heartbeat.rb

```diff
@@ -74,17 +75,12 @@ def run
   end
 
   def opsgenie_heartbeat
-    params = {}
-    params['apiKey'] = config[:api_key]
-    params['name'] = config[:name]
-
-    uri = URI.parse('https://api.opsgenie.com/v1/json/heartbeat/send')
+    encoded_name = URI.escape(config[:name])
+    uri = URI.parse("https://api.opsgenie.com/v2/heartbeats/#{encoded_name}/ping")
```

Comment:
Do we need the api key here as well?

Reply:
The API key, in the v2 API, is provided via the `Authorization` header; it is not part of the URI, nor is it a query parameter.

Reply:
makes sense.